src/deploy-cromwell-on-azure/scripts/env-03-external-images.txt

```diff
@@ -1,2 +1,2 @@
-CromwellImageName=broadinstitute/cromwell:73
```

Comment:
What was the change here?

Reply:
Whitespace, I changed something and then backed it out. I assume its the line ending but I can't seem to revert it.

Reply:
Check the line endings in the editor (in the VS IDE it's at the lower right corner of the file content box, it should be LF, not MIXED)

Reply:
Yeah, that was problem. But git won't register line ending only changes. I had to add a commit to change the file and then back it out to remove the bad line ending. Fixed now though.